src/cljs/main/broadfcui/page/workspace/method_configs/method_config_editor.cljs

```diff
@@ -225,9 +227,10 @@
          (this :-render-main locked?)]]))
    :-render-main
    (fn [{:keys [state this locals props refs]} locked?]
-     (let [{:keys [editing? loaded-config wdl-parse-error inputs-outputs entity-types methods methods-response redacted?]} @state
+     (let [{:keys [editing? loaded-config wdl-parse-error inputs-outputs entity-types methods methods-response redacted? entity-type?]} @state
            config (:methodConfiguration loaded-config)
            {:keys [methodRepoMethod rootEntityType]} config
+           entity-type-options (merge (if rootEntityType {(keyword rootEntityType) {}} {}) entity-types)
```

Comment:
You can (must) choose a root entity type when importing a method from the method repo. It would be unfriendly to disallow editing other attributes of the config while leaving root entity type alone. This allows you to keep the current root entity type selected even if the workspace has no entities of that type.